runelite-client/src/main/java/net/runelite/client/plugins/timetracking/farming/FarmingTracker.java

```diff
@@ -121,30 +128,81 @@ public boolean updateData(WorldPoint location)
 				if (storedValue != null)
 				{
 					String[] parts = storedValue.split(":");
-					if (parts.length == 2 && parts[0].equals(strVarbit))
+					if (parts.length == 2)
 					{
-						long unixTime = 0;
-						try
-						{
-							unixTime = Long.parseLong(parts[1]);
-						}
-						catch (NumberFormatException e)
+						if (parts[0].equals(strVarbit))
 						{
-							// ignored
+							long unixTime = 0;
+							try
+							{
+								unixTime = Long.parseLong(parts[1]);
+							}
+							catch (NumberFormatException e)
+							{
+								// ignored
+							}
+							if (unixTime + (5 * 60) > unixNow && unixNow + 30 > unixTime)
+							{
+								continue;
+							}
 						}
-						if (unixTime + (5 * 60) > unixNow && unixNow + 30 > unixTime)
+						else if (!newRegionLoaded)
 						{
-							continue;
+							PatchState previousPatchState = patch.getImplementation().forVarbitValue(Integer.parseInt(parts[0]));
+							PatchState currentPatchState = patch.getImplementation().forVarbitValue(client.getVar(varbit));
+							int patchTickRate = previousPatchState.getTickRate();
+
+							if (isObservedGrowthTick(previousPatchState, currentPatchState))
+							{
+								String storedOffsetPrecisionString = configManager.getRSProfileConfiguration(TimeTrackingConfig.CONFIG_GROUP, TimeTrackingConfig.FARM_TICK_OFFSET_PRECISION);
```

Comment:
```suggestion
								Integer storedOffsetPrecision = configManager.getRSProfileConfiguration(TimeTrackingConfig.CONFIG_GROUP, TimeTrackingConfig.FARM_TICK_OFFSET_PRECISION, Integer.class);
```

The config manager is capable of doing the conversion, you should use that since it handles corrupted configs.